docker/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+services:
+  roop-cpu: &base
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile.cpu
+    volumes:
+      - ../.github/examples:/data
+      - ../.cache:/cache
+    user: 1000:1000
+  roop-nvidia:
+    <<: *base
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile.nvidia
+    deploy:
+     resources:
+      reservations:
+        devices:
+          - driver: nvidia
+            count: 1
+            capabilities: [gpu]
```

Comment:
I tried if the GPU usage works in Docker with this PR and everything seems fine, except ffmpeg using hwaccel for en-/decoding.

To solve this, you need to change this line to:

```
capabilities: [gpu,video]
```
which I can confirm fixes the problem and was found [here](https://stackoverflow.com/a/75809584/4043811).

Reply:
Thank you! I've added `video` capability and that removed the warning in `ffmpeg` execution.